src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseHeaderLabel.cs

```diff
@@ -3,33 +3,55 @@
 
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
-using System.Runtime.Versioning;
 
 namespace System.Security.Cryptography.Cose
 {
+    /// <summary>
+    /// Represents a COSE header label.
+    /// </summary>
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
     public readonly struct CoseHeaderLabel : IEquatable<CoseHeaderLabel>
     {
         internal string LabelName => LabelAsString != null ? $"\"{LabelAsString}\"" : LabelAsInt32.ToString();
         private string DebuggerDisplay => $"Label = {LabelName}, Type = {(LabelAsString != null ? typeof(string) : typeof(int))}";
 
         // https://www.iana.org/assignments/cose/cose.xhtml#header-parameters
+        /// <summary>
+        /// Gets a <see cref="CoseHeaderLabel"/> that represents the known header parameter "alg".
+        /// </summary>
```

Comment:
I think most properties need a `<value>` doc comment as well. This applies to other properties throughout this PR.

```suggestion
        /// </summary>
        /// <value>
        ///  A <see cref="CoseHeaderLabel"/> that represents the known header parameter "alg".
        /// </value>
```